aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/TimeoutStepTest.java

```diff
@@ -45,27 +41,27 @@ public void killingParallel() throws Exception {
                 "node { ",
                     "timeout(time:5, unit:'SECONDS') { ",
                         "parallel(",
-                            " a: { echo 'ShouldBeHere1'; sh 'sleep 10'; echo 'NotHere' }, ",
-                            " b: { echo 'ShouldBeHere2'; sh 'sleep 10'; echo 'NotHere' }, ",
+                            " a: { echo 'ShouldBeHere1'; sleep 10; echo 'NotHere' }, ",
+                            " b: { echo 'ShouldBeHere2'; sleep 10; echo 'NotHere' }, ",
                         ");",
                         "echo 'NotHere'",
                     "}",
                     "echo 'NotHere'",
                 "}")));
-        WorkflowRun b = assertBuildStatus(Result.FAILURE, p.scheduleBuild2(0));
+        WorkflowRun b = assertBuildStatus(/* TODO JENKINS-25894 should really be ABORTED */Result.FAILURE, p.scheduleBuild2(0));
```

Comment:
Ditto above; the intention of code is for it to be `ABORTED`, but the wrapping in `ParallelStepException` breaks the transmission of this status.